06_puppy/patrickmarabeas/syncStore_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSyncStore(t *testing.T) {
+
+	t.Run("NewSyncStore", func(t *testing.T) {
+		got := NewSyncStore()
+		expected := &SyncStore{
+			uuid: 0,
+		}
+
+		if !reflect.DeepEqual(expected, got) {
+			t.Errorf("\nExpected: %#v\nGot:      %#v", expected, got)
+		}
+	})
+
+	t.Run("Create", func(t *testing.T) {
+		// Setup
+		store := NewMapStore()
+		// Create a puppy
+		id := store.Create(Puppy{Breed: "Wolf", Color: "Grey", Value: "450"})
+		expectedID := 1
```

Comment:
Is it part of the API design that the first id is 1? If so, then what should the second puppy get after inserting and removing the first Puppy? If not, then you shouldn't test for it.